### ISSUE TITLE
Unwanted blank lines between paragraphs with figures that have `placement: auto`

### ISSUE THREAD

**Issue by tshort** [flavio20002/typst-orange-template#24]
`placement: auto` floats figures up or down. When this is used, a blank line is left where the figure was created. Here's an example that reproduces the issue.

```typ
#import "@preview/orange-book:0.3.0": book, chapter
#show: book.with(
  title: "Exploring the Physical Manifestation of Humanity’s Subconscious Desires",
  subtitle: "A Practical Guide",
  date: "Anno scolastico 2023-2024",
  author: "Goro Akechi",
  main-color: rgb("#F36619"),
) 

#lorem(200)

#lorem(200)

#lorem(200)

#lorem(200)

#lorem(200)

#figure(
  table(
    columns: 4,
    [t], [1], [2], [3],
    [y], [0.3s], [0.4s], [0.8s],
  ),
  caption: [Timing results],
  placement: auto
)

#lorem(200)

#lorem(200)
```

I think the problem is in [these lines](https://github.com/flavio20002/typst-orange-template/blob/d1693c9e6392d6a46dcd6499712d5d12c9413e17/lib.typ#L318-L321). When I comment those out, the problem goes away.

**Comment by flavio20002** [comment]
Hi, right, placement auto was not implemented when I wrote the template. I will change the code in the next version with:

    show figure: it => align(center)[
      #it
      #if (it.placement == none){
        v(2.6em, weak: true)
      }
    ]

**Comment by flavio20002** [comment]
Fixed in orange-book:0.4.0
